packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -231,9 +233,17 @@ const ModalScrollContent = styled(Box)<ModalScrollContentProps>`
   height: 100%;
   width: 100%;
   max-height: ${props =>
-    !!props.FixedButton ? "calc(100vh - 172px)" : "calc(100vh - 80px)"};
+    !!props.FixedButton
+      ? "calc(100vh - (80px + " + FIXED_BUTTON_HEIGHT + "))"
+      : "calc(100vh - 80px)"};
   overflow: auto;
   padding-bottom: ${space(2)}px;
+  ${media.xs`
```

Comment:
`max-heights`' should be grouped together, and `space(9)` with an inline comment is probably fine, don't need a big constant for less than three uses

Reply:
I think the constant helps identify why I'm adding the extra space here despite it being used sparingly.